x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/action/apikey/GetApiKeyRequestTests.java

```diff
@@ -120,33 +123,59 @@ public void writeTo(StreamOutput out) throws IOException {
 
     public void testSerialization() throws IOException {
         final String apiKeyId = randomAlphaOfLength(5);
-        final boolean ownedByAuthenticatedUser = true;
-        GetApiKeyRequest getApiKeyRequest = GetApiKeyRequest.usingApiKeyId(apiKeyId, ownedByAuthenticatedUser);
         {
+            final GetApiKeyRequest getApiKeyRequest1 = GetApiKeyRequest.builder()
```

Comment:
Nit: since this is in its own scope, `getApiKeyRequest` (without the numerical suffix) still works here, and further down.